### DISCUSSION TITLE
TagArt, MusicBrainz, AlbumArt, Lyrics

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#1308] [Q&A] [answered]
I've just saw these wonderful lua scripts in your repo (great idea to seperate it. Never saw it before).

As i see some plugins needs fanart.tv subscription for key creation. There is no free usage, right? (just to be sure).

Is there a simple guide to use in order to implement the MusicBrainz to my setup?

Do i need an account to any service for Lyrics implementation?

Thank you!

**Comment by jcorporation** [comment, marked as answer]
fanart.tv needs a free account and an API key, the subscription is optional.

Script usage is documented in the README.md in the script folder. All the scripts here are working only with myMPD v16, which is not released yet.

MusicBrainz: you must tag your music with special MusicBrainz tags. I personally use Picard for tagging.

**Reply by Emilot** [reply]
> fanart.tv needs a free account and an API key, the subscription is optional.
> 
> Script usage is documented in the README.md in the script folder. All the scripts here are working only with myMPD v16, which is not released yet.
> 
> MusicBrainz: you must tag your music with special MusicBrainz tags. I personally use Picard for tagging.

Hmmm i have create an account but in order to get the API key it redirects me to "packages" about VIP stuff. I will dig some more, i missed something.

Thank you about the rest.

**Reply by jcorporation** [reply]
Direct link to get the API key: https://fanart.tv/get-an-api-key/

**Reply by Emilot** [reply]
> Direct link to get the API key: https://fanart.tv/get-an-api-key/

Thanx JC. It was a cache page issue. I talk to Discord support and i solve it (i needed to go back to first page an follow the link in the footer for second time).

I guess i can use your 16.0.0 branch, right?

**Reply by jcorporation** [reply]
You can use it, but it is an unstable development version.
